### PR TITLE
[docs] Hashicorp doc update

### PR DESCRIPTION
**Pull request by HarshDaryani896**
Updating hashicorp doc, to add instructions to directly download plugin binary.

### CODE REVIEW

File: docs/content/preview/integrations/hashicorp-vault.md [hunk 1]
```diff
@@ -67,6 +67,10 @@ Install the YSQL plugin for Hashicorp Vault as follows:
     go build -o <build dir>/ysql-plugin cmd/ysql-plugin/main.go
     ```
 
+Alternatively, you can download the binary directly from GitHub:
+
+- Pre-built binary can be found at the [releases page](https://github.com/yugabyte/hashicorp-vault-ysql-plugin/releases), download, unzip and place the binary in <build dir>.
```
Comment (ashetkar) [inline docs/content/preview/integrations/hashicorp-vault.md]:
```suggestion
- Pre-built binary can be found on the [releases page](https://github.com/yugabyte/hashicorp-vault-ysql-plugin/releases). Download, unzip the file and place the binary in <build dir>.
```

Comment (ashetkar) [reply docs/content/preview/integrations/hashicorp-vault.md]: line # 32:
`- Add go to the path as follows:` -> `- Add Go to the path as follows:`

Comment (aishwarya24) [reply docs/content/preview/integrations/hashicorp-vault.md]: Download, unzip the file, and place the binary in <build dir>.

Comment (aishwarya24) [reply docs/content/preview/integrations/hashicorp-vault.md]: @HarshDaryani896 LGTM with the suggestions. Please make the changes and merge.

**Review by aishwarya24 — APPROVED** [review]

**Pull request merged**